src/fstbin/fsts-project.cc

```diff
@@ -0,0 +1,80 @@
+// fstbin/fsts-project.cc
+
+// Copyright 2016  Johns Hopkins University (Authors: Jan "Yenda" Trmal)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include "base/kaldi-common.h"
+#include "util/common-utils.h"
+#include "fstext/fstext-utils.h"
+#include "fstext/kaldi-fst-io.h"
+
+
+int main(int argc, char *argv[]) {
+  try {
+    using namespace kaldi;
+    using namespace fst;
+    typedef kaldi::int32 int32;
+    typedef kaldi::uint64 uint64;
+
+    const char *usage =
+        "Reads a table of FSTs; for each element, performs the fst project\n"
+        "operation either on input (default) or on the output (if the option\n"
+        "--project-output is true).\n"
+        "\n"
+        "Usage: fsts-project [options] <fsts-rspecifier> <fsts-wspecifier>\n"
+        " e.g.: fsts-project ark:train.fsts ark,t:train.fsts\n";
```

Comment:
can we have a "see also:" line?  Like:
"see also: fstproject (from the OpenFst toolkit)"
and the same for similar tools?